package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -281,15 +281,15 @@ def update_dictionary_point_in_cube_start_frame(array_simulation_particle_coordi
                                                            axis=0)
                 cube['centroid_of_particles_first_frame'] = centroid_particles_in_cube
             else:  # empty cube
-                cube['centroid_of_particles_first_frame'] = 'empty'
+                cube['centroid_of_particles_first_frame'] = None
             cube_counter += 1
 
     def update_dictionary_end_frame(array_simulation_particle_coordinates, dictionary_cube_data_this_core):
         """Update the cube dictionary objects again as appropriate for the second and final frame."""
         cube_counter = 0
         for key, cube in dictionary_cube_data_this_core.items():
             # if there were no particles in the cube in the first frame, then set dx,dy,dz each to 0
-            if cube['centroid_of_particles_first_frame'] == 'empty':
+            if cube['centroid_of_particles_first_frame'] == None:
```

Comment:
That took me far too long to narrow down, looks like the issue here was that we are at times doing an array vs str comparison, which until 1.24 led to a scalar return, but now returns an array of bools instead, that in turns messes up this if/else call.

Doing a comparison against `None` is probably just safer & more pythonic here?

P.S. Weirdly enough I think this should have thrown an error rather than silently failing i.e. it _should_ have complained about the truth of an array being ambiguous.. but it didn't 😕

Reply:
See https://github.com/MDAnalysis/mdanalysis/actions/runs/3824585204/jobs/6506856469 for a successful run.